openrtx/include/core/thread_utils.h

```diff
@@ -0,0 +1,262 @@
+/**
```

Comment:
For each of these new `openrtx/**/*utils.(h|c)` files, please move them to a separate PR (the same as referenced in my previous comment).